docs/cedarling/cedarling-policy-store.md

```diff
@@ -13,36 +13,39 @@ tags:
 The Cedarling Policy Store uses a JSON file named `cedarling_store.json` to store all necessary data for evaluating policies and verifying JWT tokens. The structure includes the following key components:
 
 1. **Cedar Schema**: The Cedar schema encoded in Base64.
-2. **Cedar Policies**: The Cedar policies encoded in Base64.
+2. **Policies**: The Cedar policies encoded in Base64.
 3. **Identity Source**: Details about the trusted issuers (see [below](#identity-source-schema) for syntax).
 
+You could either define your own schema or generate one using the Agama Lab Policy Designer. See: [Policy and Schema Authoring](#policy-and-schema-authoring).
+
 **Note:** The `cedarling_store.json` file is only needed if the bootstrap properties: `CEDARLING_LOCK`; `CEDARLING_POLICY_STORE_URI`; and `CEDARLING_POLICY_STORE_ID` are not set to a local location. If you're fetching the policies remotely, you don't need a `cedarling_store.json` file.
 
-### JSON Schema
+## Cedarling Store Schema
 
 The JSON Schema accepted by cedarling is defined as follows:
 
 ```json
 {
   "cedar_version": "v4.0.0",
-  "policy_stores": {
-      "some_unique_string_id": {
-          "name": "TestPolicy",
-          "description": "Once upon a time there was a Policy, that lived in a Store.",
-          "policies": { ... },
-          "schema": { ... },
-          "identity_source": { ... }
-      }
-  }
+  "name": "Jans::Store",
```

Comment:
Name only `Jans` and it is defined the `agama-lab`
and looks like you need to revert this change...